article/app/views/fragments/articleBodyHeroic.scala.html

```diff
@@ -28,9 +28,9 @@
                 <div class="gs-container">
                     <div class="content__main-column content__main-column--article js-content-main-column">
                         <div class="content__article-body from-content-api js-article__body" itemprop="articleBody">
-                            @if(article.content.tags.isLabourLiverpoolSeries) {
-                                <img class="labour-liverpool-cutout hide-on-mobile" src="@ImgSrc("https://static.guim.co.uk/sys-images/Guardian/Pix/contributor/2015/8/18/1439913873894/Ewen-MacAskill-R.png", Item140)">
-                                @fragments.inlineSvg("quote", "icon", List("labour-liverpool-quotes"))
+                            @if(article.content.tags.isExploreSeries) {
```

Comment:
is the `isExploreSeries` different from the "Heroic template"? If not, this file should be renamed and there shouldn't be a need for this if statement